8.8 minor release notes

mesellings opened the pull request:
## Description

8.8 minor release notes.

<!-- Provide an overview of what to expect in the PR. -->
<!-- Relate or link the associated epic or task. -->
<!-- Add `@camunda/tech-writers` as reviewer to pull in a tech writer, or add your embedded tech writer. -->

## When should this change go live?

<!-- PRs merged go to stage.docs.camunda.io first and must be manually released to docs.camunda.io. -->

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**. (add `bug` or `support` label)
- [ ] This is already available but undocumented and should be released within a week. (add `available & undocumented` label)
- [ ] This is on a **specific schedule** and the assignee will coordinate a release with the Documentation team. (create draft PR and/or add `hold` label)
- [X] This is part of a scheduled **alpha or minor**. (add alpha or minor label)
- [ ] There is **no urgency** with this change (add `low prio` label)

## PR Checklist

- [ ] The commit history of this PR is cleaned up, using [`{type}(scope): {description}` commit message(s)](https://github.com/camunda/camunda-docs/blob/main/CONTRIBUTING.MD#commit-message-header-formatting)

<!-- Camunda maintains 18 months of minor versions. Backporting your change to multiple versions is common. -->

- [X] My changes are for **an upcoming minor release** and are in the `/docs` directory (version 8.8).
- [ ] My changes are for an **already released minor** and are in a `/versioned_docs` directory.

<!-- UNCOMMENT THIS SECTION IF APPLICABLE. Adding or removing pages requires extra steps.
- [ ] I included my new page in the sidebar file(s).
- [ ] I added a redirect for a renamed or deleted page to the .htaccess file.
-->

- [ ] I added a DRI, team, or delegate as a reviewer for technical accuracy and grammar/style:
  - [ ] [Engineering team review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process)
  - [X] [Technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process) via `@camunda/tech-writers` unless working with an embedded writer.

<!-- UNCOMMENT THIS SECTION IF APPLICABLE. Changes to **docs infra**, including updates to workflows and adding new npm packages, must be first discussed via issue or #ask-c8-documentation and linked for context.
- [ ] My changes require a [docs infrastructure review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process). (add `dx` label) -->

mesellings commented:
FYI @MaxTru @megglos @theburi, I've tagged you in as reviewers to get your feedback on the new release announcements page - I've tidied it up and tried to put things into the relevant places, but would appreciate a check with what I've done as a first pass - I haven't tidied up wording or anything yet. Changes are made to make it clearer for users to easily see/scan the key/breaking changes section in particular.

Preview environment is here: https://preview.docs.camunda.cloud/pr-6802/docs/next/reference/announcements-release-notes/880/880-release-notes/

p.s. Don't worry about the release notes - I will post in #team-products soon about these, and they are still WIP.

MaxTru reviewed (approved):
Thank you. Outstanding overview page.

Approving (even though there will be at least one additional entry => @hisImminence will contribute an announcement about deprecation of the legacy docker images (camunda/tasklist, etc))). This additional entry can be added as part of a follow-up PR.

mesellings commented:
> Thank you. Outstanding overview page.
> 
> Approving (even though there will be at least one additional entry => @hisImminence will contribute an announcement about deprecation of the legacy docker images (camunda/tasklist, etc))). This additional entry can be added as part of a follow-up PR.

Thanks - I'll also be adding breaking changes info about the unified configuration as well at some point

MaxTru commented:
@mesellings I added automation so that all technical changelogs will be linked (once available) with this commit: https://github.com/camunda/camunda-docs/pull/6802/commits/b6f800c395ae934e461dfe13179bc83248f4fd25

vsgoulart commented:
@MaxTru @mesellings Please check my commit which documents the Tasklist UI versions modes

megglos commented on docs/reference/announcements-release-notes/880/880-announcements.md:
```diff
@@ -196,182 +402,287 @@ connector.context()
+</div>
+</div>
+
+<!-- ### Camunda Exporter <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
```
🔧 is there a follow-up issue to bring this back?

mesellings replied:
Apologies @megglos I commented out while I was reorganizing the page - this will be added back in in this PR, minus all the diagrams and content not needed.

mesellings replied:
Done in the latest commit

megglos reviewed (approved):
Looks good to me generally, I checked mostly API/identity specific matters. Nothing blocking besides one question on a commented out exporter announcement.

vsgoulart commented:
> @vsgoulart I've modified this on reflection, as we want to push people away from using the v1 UI - I've removed the entry but added in a note to reference the API versions page, and added the information onto that page instead.

@mesellings I was working with Volodymyr on improvements on the Tasklist version docs on [this PR](https://github.com/camunda/camunda-docs/pull/6902) to not pollute this PR too much
Maybe we should move it to there?

mesellings commented:
> > @vsgoulart I've modified this on reflection, as we want to push people away from using the v1 UI - I've removed the entry but added in a note to reference the API versions page, and added the information onto that page instead.
> 
> @mesellings I was working with Volodymyr on improvements on the Tasklist version docs on [this PR](https://github.com/camunda/camunda-docs/pull/6902) to not pollute this PR too much Maybe we should move it to there?

Thanks @vsgoulart - yep that sounds like a good plan 👍
